src/cli/add.rs

```diff
@@ -218,8 +216,13 @@ pub async fn add_pypi_specs_to_project(
             }
         }
     }
+    let lock_file_usage = if no_update_lockfile {
+        LockFileUsage::Frozen
+    } else {
+        LockFileUsage::Update
+    };
```

Comment:
Could we pass to lockfileusage to this function as well?

Reply:
Sure!